Makefile

```diff
@@ -38,6 +38,7 @@ integration-test:
 	@echo "=== $(INTEGRATION) === [ test ]: running integration tests..."
 	@docker-compose -f tests/integration/docker-compose.yml up -d --build
 	@go test -tags=integration ./tests/integration/. || (ret=$$?; docker-compose -f tests/integration/docker-compose.yml down && exit $$ret)
+	@go test -tags=integration ./tests/integration/.  --host=redis-tls --enable_tls=true  || (ret=$$?; docker-compose -f tests/integration/docker-compose.yml down && exit $$ret)
```

Comment:
I would have personally added another function in the integration test with func TestRedisIntegrationWithTLS(t *testing.T) adding those flags and pasing those flags and testing only the all call with TLS, because I think then it's more clear and with only testing that TLS connection works with one case it's enough. But as you wish.

Reply:
the problem is that hostname for example was a flag in the first place, we need to refactor a bit tests.
But you are right, we should not test twice things using TLS

Reply:
I'll refactor that